CRM/Report/Form/Contribute/PCP.php

```diff
@@ -166,6 +166,27 @@ public function __construct() {
         ),
         'grouping' => 'pcp-fields',
       ),
+      'civicrm_financial_trxn' => array(
+        'dao' => 'CRM_Financial_DAO_FinancialTrxn',
+        'fields' => array(
+          'card_type' => array(
+            'title' => ts('Credit Card Type'),
+            'dbAlias' => 'GROUP_CONCAT(financial_trxn_civireport.card_type SEPARATOR ",")',
+          ),
+        ),
+        'filters' => array(
+          'card_type' => array(
+            'title' => ts('Credit Card Type'),
+            'operatorType' => CRM_Report_Form::OP_MULTISELECT,
+            'options' => CRM_Core_PseudoConstant::get('CRM_Financial_DAO_FinancialTrxn', 'card_type'),
```

Comment:
@eileenmcnaughton I didn't find any BAO that represents ```FinancialTrxn``` entity or are you suggesting that there need to add ```CRM_Financial_DAO_FinancialTrxn::getOptions(...)``` function and we should call it after that ?